torch/fx/traceback.py

```diff
@@ -31,6 +32,20 @@ def set_stack_trace(stack : List[str]):
     if should_preserve_node_meta and stack:
         current_meta["stack_trace"] = "".join(stack)
 
+@compatibility(is_backward_compatible=False)
+def set_bwd_seq_id(max_fwd_seq_id):
```

Comment:
reminder to myself to check this again to make sure the globals are fine

Reply:
I kept set_bwd_seq_id() but it no longer uses global variables.